tools/sgen/input/FluentUIStyle.yml

```diff
@@ -445,6 +444,19 @@ AP_MSFListCellTokens:
   sublabelFont: $Typography.caption1
   trailingItemForegroundColor: $Colors.Foreground.neutral3
   trailingItemSize: $Icon.size.medium
+  
+AP_MSFHeaderFooterTokens:
+  backgroundColor: [ default: $Colors.Background.neutral1, divider: $Colors.Background.neutral5 ]
+  headerHeight: [ default: $Spacing.xxxlarge, divider: $Spacing.xlarge ]
+  horizontalHeaderPadding: $Spacing.medium
```

Comment:
You can replace **horizontalHeaderPadding** and **topHeaderPadding** with a "padding" token that would be like:

`padding: Insets($Spacing.xlarge, $Spacing.medium, $Spacing.medium, $Spacing.medium)`